### PR TITLE
Add Hangeul Cheat Sheet

### PR DESCRIPTION
**Pull request by julambl75**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Hangeul Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Hangeul lists all of the letters that comprise the Korean alphabet (hangeul/hangul), along with their names and pronunciation. These letters can be assembled in order to read and write in Korean.

###### Which issues (if any) does this fix?

Correct ID

Appended "_cheat_sheet" to name.

###### People to notify (@mention interested parties)

@daxtheduck 

---

Instant Answer Page: https://duck.co/ia/view/hangeul_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@julambl75 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
thanks @julambl75 - we'll have to see why the checks are failing.  we can review this soon.

**Comment by julambl75** [comment]
The corresponding IA Page is still in the "planning" stage, it should move to the "development" stage soon (as it says in the check).

**Comment by julambl75** [comment]
It appears to work correctly using DuckPAN Server on Codio.

**Comment by julambl75** [comment]
**hangeul** failed the test because there is an AI called *hangeul* (which was previously the AI that was being used), whose name is a subset of the name of the currently used AI (*hangeul_cheat_sheet*). This had to be done because **Terminal** "said" that the AI had to be called "*hangeul_cheat_sheet*".

Now, the two AIs coexist and there is **no apparent way to delete the unnecessary AI**.

```
# Testing hangeul.................. 1 FAILURE                                                                                                                                   
#        -> FAIL: aliases contain the cheat sheet name (hangeul)                                                                                                                
                                                                                                                                                                                
    #   Failed test 'FAIL: aliases contain the cheat sheet name (hangeul)'                                                                                                      
    #   at t/CheatSheets/CheatSheetsJSON.t line 288.                                                                                                                            
    # Looks like you failed 1 test of 1.                                                                                                                                        
                                                                                                                                                                                
#   Failed test 'can_build'                                                                                                                                                     
#   at t/CheatSheets/CheatSheetsJSON.t line 289.
```

**Comment by GuiltyDolphin** [comment]
You need to remove the 'hangeul' alias.

**Comment by julambl75** [comment]
@GuiltyDolphin Thank you, I had forgotten to look inside the JSON file for the **alias** option.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=hangeul cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/hangeul_cheat_sheet_mobile.png?1Ld5d37y4o)

**Comment by edgesince84** [comment]
thanks for the help @GuiltyDolphin  - @julambl75 we'll be sharing this by the end of the week so we can get some language review!  thanks. 💃

**Comment by julambl75** [comment]
@edgesince84, thank you for helping with this cheat sheet.

**Comment by edgesince84** [comment]
you're welcome!

File: share/goodie/cheat_sheets/json/hangeul.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    "id": "hangeul_cheat_sheet",
+    "name": "Hangeul/Hangul",
+    "description": "Letters of the Korean Alphabet",
+
+    "metadata": {
+        "sourceUrl" : "https://en.wikipedia.org/wiki/Hangul"
+    },
+
+    "aliases": [
+        "hangul", "korean alphabet", "korean language", "choson gul", "choson'gul", "choson muntcha", "choson muncha"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/hangeul.json]: Please could you remove "korean language" from this list, as we'll hopefully have a cheat sheet for Korean language phrases in future.

File: share/goodie/cheat_sheets/json/hangeul.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
+{
+    "id": "hangeul_cheat_sheet",
+    "name": "Hangeul/Hangul",
+    "description": "Letters of the Korean Alphabet",
+
+    "metadata": {
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/hangeul.json]: Please could you add

    "sourceName": "Wikipedia",

to this `metadata` object?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=hangeul cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/hangeul_cheat_sheet_mobile.png?gc7XpMyAsP)

**Comment by tagawa** [comment]
Hi @julambl75. Thanks for this cheat sheet. I've just left a couple of comments before we ask for a final review from the community.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=hangeul cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/hangeul_cheat_sheet_mobile.png?UMjFU4fE2j)

**Comment by julambl75** [comment]
@edgesince84, the 2 changes have been made.
@tagawa, sorry for the issues and thank you for the help.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=hangeul cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/hangeul_cheat_sheet_mobile.png?TBzCAqoHBE)

**Comment by tagawa** [comment]
Thanks for the quick edits @julambl75 and no need to apologise!
I've just put a request on Twitter to see if there are any further comments: https://twitter.com/duckduckgo/status/764068151919206401

**Comment by unascribed** [comment]
*Not* a Korean language expert, but Consonants seems to be typo'd as Consonanants.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=hangeul cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/hangeul_cheat_sheet_mobile.png?cGByi6It7k)

**Comment by tagawa** [comment]
@unascribed Thank you very much for reviewing and commenting.

@julambl75 No need to apologise - this is why we have reviews! I'll leave it open for a couple more days and if no more feedback, we should be good to merge 😃

**Comment by tagawa** [comment]
@julambl75 We had a comment on Twitter that the ordering should be changed:

> except for the ordering.
> the way Korean remind of is
> ㄱㄴㄷㄹㅁㅂㅅㅇㅈㅊㅋㅌㅍㅎ ㅏㅑㅓㅕㅗㅛㅜㅠㅡㅣ
> The others are double/triple vowels.

https://twitter.com/ShitInTolerance/status/764450228858712064

Is this something you'd be able to update?

**Comment by julambl75** [comment]
@tagawa, yes of course.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=hangeul cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/hangeul_cheat_sheet_mobile.png?fkLuOZFPPk)

**Comment by tagawa** [comment]
Well no further review comments so let's merge this. Thank you very much @julambl75 for creating and updating this so quickly! It should go live in a few days time 🎉

**Comment by daxtheduck** [comment]
@julambl75 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by julambl75** [comment]
Thank you very much for all your help @tagawa.

**Pull request merged**